assets/prefabs/trunkRegion.prefab

```diff
@@ -0,0 +1,118 @@
+{
+    "DisplayName": {
+        "name": "Trunk"
+    },
+    "Trunk": {
+        "leftBlockFamily": "CoreBlocks:TrunkLeft",
+        "rightBlockFamily": "CoreBlocks:TrunkRight",
+        "openSound": "CoreAssets:DoorOpen",
+        "closeSound": "CoreAssets:DoorClose"
+    },
+    "Health": {
+        "currentHealth": 5,
+        "maxHealth": 5,
+        "regenRate": 1,
+        "waitBeforeRegen": 2,
+        "destroyEntityOnNoHealth": true
+    },
+    "BlockDropGrammar": {
+        "itemDrops": [
+            "CoreBlocks:trunk"
+        ]
+    },
+    "ActAsBlock": {
+        "block": "CoreBlocks:TrunkLeft"
+    },
+    "InventoryAccess": {
+        "input": {
+            "top": "0..59",
+            "bottom": "0..59",
+            "left": "0..29",
+            "right": "30..59",
+            "front": "0..59",
+            "back": "0..59"
+        },
+        "output": {
+            "top": "0..59",
+            "bottom": "0..59",
+            "left": "0..29",
+            "right": "30..59",
+            "front": "0..59",
+            "back": "0..59"
+        }
+    },
+    "Inventory": {
+        "privateToOwner": false,
+        "itemSlots": [
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0
+        ]
+    },
+    "RetainBlockInventory": {},
+    "PlaySoundAction": {
+        "sounds": "engine:click"
```

Comment:
Why this sound action? I though opening and closing were their own sounds? Or is this about the inventory interaction?

Reply:
I copied it from the chest item; we just don't have separate sounds for that yet.